### PR TITLE
Rename Image Archive to Media Archive

### PR DESCRIPTION
**Pull request by comzeradd**
As per refined requirements. Changing the most import things and UI elements.

I didn't go as far as renaming every single function or piece of code. Just the things that would create some confusion when we resume looking into that code. Simplified also the menu options. Maybe we can add the "API Info" in the credentials page if we think it's necessary.

---

Ref: https://jira.greenpeace.org/browse/PLANET-5715

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [telesto](https://www-dev.greenpeace.org/test-telesto) | [admin](https://www-dev.greenpeace.org/test-telesto/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-telesto/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-telesto) | [composer-local.json](https://github.com/greenpeace/planet4-test-telesto/blob/main/composer-local.json)

:watch: 2023.05.24 07:58:40

File: src/ImageArchive/UiIntegration.php [hunk 1]
```diff
@@ -1,144 +0,0 @@
...
-            [ self::class, 'api_info' ]
-        );
-    }
-
-    /**
-     * Display information about the media API.
-     */
-    public static function api_info(): void
-    {
-        Loader::enqueue_versioned_style('/admin/css/media-api-info.css');
```
Comment (sagarsdeshmukh) [inline src/ImageArchive/UiIntegration.php:102]: We are no more needed `/admin/css/media-api-info.css` file as well.

File: src/MediaArchive/UiIntegration.php [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    public static function picker_page(): void
+    {
+        if (! current_user_can(Capability::USE_IMAGE_ARCHIVE_PICKER)) {
+            return;
+        }
+
+        add_menu_page(
+            __('Media Archive', 'planet4-master-theme-backend'),
+            __('Media Archive', 'planet4-master-theme-backend'),
+            Capability::USE_IMAGE_ARCHIVE_PICKER,
```
Comment (sagarsdeshmukh) [inline src/MediaArchive/UiIntegration.php:64]: Last instance of image archive left 🙂

**Comment by sagarsdeshmukh** [comment]
Tested the PR on local, looks good to me 👍

**Review by sagarsdeshmukh — APPROVED** [review]

**Pull request merged**